priv/templates/nimble_template/.github/workflows/publish_wiki.yml.eex

```diff
@@ -0,0 +1,51 @@
+name: Publish Wiki
+
+on:
+ push:
+    paths:
+      - .github/wiki/**
+    branches:
+      - develop
+
+env:
+  USER_TOKEN: ${{ secrets.GH_TOKEN }}
+  USER_NAME: github-wiki-workflow
+  USER_EMAIL: dev@nimblehq.co
```

Comment:
Should we hardcode this as `dev@nimblehq.co` @bterone 🤔 as the template is for the community, how about we change it to something general 🤔 Eg: `EMAIL_ADDESS`

Reply:
I'm thinking we can keep it consistent with `GH_TOKEN` and call it `GH_EMAIL`, what do you think? @andyduong1920 🤔

Reply:
That works @bterone

Reply:
Adjusted in 07e34db910664483d24974a75e9a79f0951e6975 👍